### ISSUE TITLE
Should CHEMBL.TARGET:CHEMBL4530 be an equivalent curie with UniProtKB:P00488?

### ISSUE THREAD

**Issue by chunyuma** [RTXteam/RTX#1350]
Hi Eric (@edeutsch), could you please take a look why some curie ids with prefix `CHEMBL.TARGET` can't be clustered with `UniProtKB` in kg2c-5-2? Currently, there are 2315 curie ids with prefix `CHEMBL.TARGET` and 25203 curie ids with prefix `UniProtKB

![Screen Shot 2021-04-07 at 3 02 11 PM](https://user-images.githubusercontent.com/20031672/113920029-4014a600-97b2-11eb-9876-8bf98a6200c5.png)
`
### Here are a few examples:

`CHEMBL.TARGET:CHEMBL4530` and `UniProtKB:P00488`:

Based on ChEMBL website [here](https://www.ebi.ac.uk/chembl/target_report_card/CHEMBL4530/), `CHEMBL.TARGET:CHEMBL4530` should be an equivalent curie with `UniProtKB:P00488`.

![Screen Shot 2021-04-07 at 3 10 29 PM](https://user-images.githubusercontent.com/20031672/113920952-70a90f80-97b3-11eb-9167-c7fa5662fb05.png)

![Screen Shot 2021-04-07 at 3 10 43 PM](https://user-images.githubusercontent.com/20031672/113920958-7272d300-97b3-11eb-93ca-3c77be90550b.png)

`CHEMBL.TARGET:CHEMBL3295` and `UniProtKB:P0DJD9`:

![Screen Shot 2021-04-07 at 3 08 07 PM](https://user-images.githubusercontent.com/20031672/113920667-1445f000-97b3-11eb-95f0-3f749e0d7fc6.png)

![Screen Shot 2021-04-07 at 3 08 58 PM](https://user-images.githubusercontent.com/20031672/113920804-43f4f800-97b3-11eb-8505-794c6eaab46a.png)

`CHEMBL.TARGET:CHEMBL2802` and `UniProtKB:P43153`:

![Screen Shot 2021-04-07 at 3 06 24 PM](https://user-images.githubusercontent.com/20031672/113920489-d779f900-97b2-11eb-8824-1548c165f000.png)

![Screen Shot 2021-04-07 at 3 07 27 PM](https://user-images.githubusercontent.com/20031672/113920612-fd070280-97b2-11eb-905e-f51bee18a8ad.png)

**Comment by edeutsch** [comment]
Thanks for the report. following up on the first one:
```
$ egrep 'CHEMBL.TARGET:CHEMBL4530|UniProtKB:P00488' kg2_node_info.tsv
UniProtKB:P00488	F13A1	Coagulation factor XIII A chain	biolink:Protein
CHEMBL.TARGET:CHEMBL4530	Coagulation factor XIII	Coagulation factor XIII	biolink:Protein

$ egrep 'CHEMBL.TARGET:CHEMBL4530|UniProtKB:P00488' kg2_equivalencies.tsv 
$
```

Asking the SRI Node Normalizer:
https://nodenormalization-sri.renci.org/get_normalized_nodes?curie=CHEMBL.TARGET:CHEMBL4530
null

https://nodenormalization-sri.renci.org/get_normalized_nodes?curie=UniProtKB:P00488
```
{"UniProtKB:P00488":{"id":{"identifier":"NCBIGene:2162","label":"F13A1"},"equivalent_identifiers":[{"identifier":"NCBIGene:2162"},{"identifier":"ENSEMBL:ENSG00000124491"},{"identifier":"HGNC:3531","label":"F13A1"},{"identifier":"UniProtKB:P00488","label":"F13A_HUMAN Coagulation factor XIII A chain"}],"type":["biolink:Gene","biolink:GenomicEntity","biolink:MolecularEntity","biolink:BiologicalEntity","biolink:NamedThing","biolink:Entity"]}}
```

So basically, they are not in equivalencies and the SRI Node Normalizer does not link them.
The only other way to link them is by name, and their names are:

Coagulation factor XIII A chain
Coagulation factor XIII

I could add a manual rule that for all UniProtKB proteins that are "xxxxxxxxxx A chain" they get a synonym without the "A chain" as well. But would that break other things? (i.e. link things we don't want?)

Here's a bit more info:
```
$ egrep 'Coagulation factor XIII' kg2_node_info.tsv
...
SNOMED:775331003	Coagulation factor XIII only product		biolink:Drug
SNOMED:9532005	Coagulation factor XIIIa		biolink:ChemicalSubstance
SNOMED:51161000	Coagulation factor XIII		biolink:ChemicalSubstance
SNOMED:319930009	Coagulation factor XIII-containing product		biolink:Drug
UniProtKB:P00488	F13A1	Coagulation factor XIII A chain	biolink:Protein
UniProtKB:P05160	F13B	Coagulation factor XIII B chain	biolink:Protein
CHEMBL.TARGET:CHEMBL4530	Coagulation factor XIII	Coagulation factor XIII	biolink:Protein
CHEMBL.TARGET:CHEMBL3351193	Coagulation factor XIII B chain	Coagulation factor XIII B chain	biolink:Protein
PathWhiz.ProteinComplex:144	Coagulation factor XIII A chain	Coagulation factor XIII A chain	biolink:MolecularEntity
PathWhiz.ProteinComplex:145	Coagulation factor XIII B chain	Coagulation factor XIII B chain	biolink:MolecularEntity
PathWhiz.ProteinComplex:638	Coagulation factor XIIIa	Coagulation factor XIIIa	biolink:MolecularEntity
PathWhiz.ProteinComplex:11727	Coagulation factor XIIIa	Coagulation factor XIIIa	biolink:MolecularEntity
PathWhiz.ProteinComplex:11725	Coagulation factor XIII A chain	Coagulation factor XIII A chain	biolink:MolecularEntity
PathWhiz.ProteinComplex:11726	Coagulation factor XIII B chain	Coagulation factor XIII B chain	biolink:MolecularEntity
PathWhiz.ProteinComplex:11569	Coagulation factor XIIIa	Coagulation factor XIIIa	biolink:MolecularEntity
PathWhiz.ProteinComplex:11567	Coagulation factor XIII A chain	Coagulation factor XIII A chain	biolink:MolecularEntity
PathWhiz.ProteinComplex:11568	Coagulation factor XIII B chain	Coagulation factor XIII B chain	biolink:MolecularEntity
PathWhiz.ProteinComplex:11648	Coagulation factor XIIIa	Coagulation factor XIIIa	biolink:MolecularEntity
PathWhiz.ProteinComplex:11646	Coagulation factor XIII A chain	Coagulation factor XIII A chain	biolink:MolecularEntity
PathWhiz.ProteinComplex:11647	Coagulation factor XIII B chain	Coagulation factor XIII B chain	biolink:MolecularEntity
```

It's not clear what the right path here is.

Is it important for the purposes of ARAX to distinguish between "A chain" proteins and "B chain" proteins? By giving them all a synonym without the "X chain" they would get clustered together. I am uncertain if that is a wise decision.

**Comment by edeutsch** [comment]
Or to fix this one case I can manually rename CHEMBL.TARGET:CHEMBL4530	
from Coagulation factor XIII
to Coagulation factor XIII A chain

that would also cause them to link.
But that would only fix this one case and there are more.

**Comment by chunyuma** [comment]
Thanks @edeutsch. It seems like it is not a good decision that adding a manual rule that for all UniProtKB proteins that are "xxxxxxxxxx A chain" they get a synonym without the "A chain" as well because this might be just one of some "not clustered" cases. As you can see in other cases that I posted, it seems like most `CHEMBL.TARGET` nodes (possibly all of them) will have a corresponding `UniProtKB ` id if our KG2 includes all `UniProtKB ` ids. 

I don't know if finally the SRI Node Normalizer can solve this problem.

**Comment by edeutsch** [comment]
Another possible interpretation is that the CHEMBL.TARGETs are really referring to protein complexes, while UniProtKB entries are individual proteins. So perhaps they really are different concepts. Perhaps often the individual proteins are "A chain", "B chain" of a protein complex and some concepts in KG2 are really the complexes.

There is a data source called the Complex Portal at EBI. I wonder if it might be advantageous to consider ingesting parts of the Complex Portal into KG2? @saramsey @ericawood @kvarforl ?
https://www.ebi.ac.uk/complexportal/complex/search?query=coagulation%20factor%20XIII&page=1
I saw a presentation recently by the head curators and it seems like a vary carefully curated and high quality resource.

Although even if we did, it's not clear this would solve the problem at hand in this issue.
